docs/Learning-Environment-Design-Agents.md

```diff
@@ -174,11 +240,10 @@ observes that the current item is a Bow, you would add the elements: 0, 0, 1 to
 the feature vector. The following code example illustrates how to add.
 
 ```csharp
-enum CarriedItems { Sword, Shield, Bow, LastItem }
-private List<float> state = new List<float>();
+enum ItemType { Sword, Shield, Bow, LastItem }
```

Comment:
Bike-shedding a bit, but I didn't like the old name. I still don't love having `LastItem` in the enum.